servers/quarkus-server/src/test/java/org/projectnessie/server/catalog/AbstractIcebergCatalogUnitTests.java

```diff
@@ -205,4 +215,52 @@ public void namespacesPaging() throws Exception {
                 .mapToObj(i -> TableIdentifier.of("namespace_0", "view_" + i))
                 .toList());
   }
+
+  @Test
```

Comment:
You can possibly tweak the object-storage-mock to return failures to test.

Reply:
What do you mean? The object storage mock is already producing failures when the test callbacks instruct it to.